prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2587,59 +2616,59 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
-  - name: metal3-ubuntu-e2e-feature-test-release-1-5-pivoting
+  # name: {job_prefix}-e2e-clusterctl-upgrade-test-{capm3_target_branch}
+  - name: metal3-e2e-clusterctl-upgrade-test-main
     branches:
-    - release-1.5
+    - main
     agent: jenkins
     always_run: false
     optional: true
-  - name: metal3-ubuntu-e2e-feature-test-release-1-5-remediation
+  - name: metal3-e2e-clusterctl-upgrade-test-release-1-8
     branches:
-    - release-1.5
+    - release-1.8
     agent: jenkins
     always_run: false
     optional: true
-  - name: metal3-ubuntu-e2e-feature-test-release-1-5-features
+  - name: metal3-e2e-clusterctl-upgrade-test-release-1-7
     branches:
-    - release-1.5
+    - release-1.7
     agent: jenkins
     always_run: false
     optional: true
-  # name: {job_prefix}-e2e-clusterctl-upgrade-test-{capm3_target_branch}
-  - name: metal3-e2e-clusterctl-upgrade-test-main
+  # name: {job_prefix}-e2e-{k8s_versions}-upgrade-test-{capm3_target_branch}
+  - name: metal3-e2e-1-28-1-29-upgrade-test-main
```

Comment:
Please double check what versions are going to be tested in the new main upgrade